tests/flow/test_procedures.py

```diff
@@ -371,6 +371,24 @@ def test12_procedure_reordered_yields(self):
                            ["READ",  "db.labels"],
                            ["READ",  "db.propertyKeys"],
                            ["READ",  "db.relationshipTypes"],
+                           ["READ",  "dbms.functions"],
                            ["READ",  "dbms.procedures"]]
         self.env.assertEquals(actual_resultset, expected_result)
 
+    def test13_functions(self):
+        # Yield results of procedure in a non-default sequence
+        actual_resultset = self.graph.query("CALL dbms.functions() YIELD name RETURN name ORDER BY name").result_set
+        actual_len = len(actual_resultset)
+        self.env.assertEquals(actual_len, 95)
+
+    def test14_procedure_get_all_functions(self):
+        actual_resultset = self.graph.query("CALL dbms.functions() YIELD name, signature, description, return_description, argument_description RETURN name, signature, description, return_description, argument_description ORDER BY name").result_set
```

Comment:
consider renaming `return_description` to `return type(s)`, `argument_description` -> `argument(s)_type`
Also I think the last two arguments are a bit redundant given that we have `signature` unless we provide additional verbal description to the arguments / return type.

Reply:
renaming done, the last 2 output was requested by Avi after checking Neo4j output